asab/api/discovery.py

```diff
@@ -312,3 +402,19 @@ async def resolve(self, hostname: str, port: int = 0, family: int = socket.AF_IN
 
 class NotDiscoveredError(RuntimeError):
 	pass
+
+
+def _generate_private_key() -> bytes:
+	import cryptography.hazmat.backends
```

Comment:
This will introduce a dependency on cryptography module?

Reply:
i can make it optional, we already have it like that in [setup.py](https://github.com/TeskaLabs/asab/blob/master/setup.py#L83)
the dependency is already there in [asab.web.webcrypto](https://github.com/TeskaLabs/asab/blob/master/asab/web/webcrypto.py#L4) and **optionally** in [asab.storage](https://github.com/TeskaLabs/asab/blob/master/asab/storage/service.py#L9)

Reply:
Does `jwtcrypto` provide a key generation functions?

Reply:
discussed on review